extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/utils/EdcPropertyUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *      sovity GmbH - init
+ */
+
+package de.sovity.edc.ext.wrapper.utils;
+
+import lombok.NoArgsConstructor;
+import org.eclipse.edc.spi.types.domain.DataAddress;
+
+import java.util.Map;
+
+import static de.sovity.edc.ext.wrapper.utils.MapUtils.mapValues;
+
+@NoArgsConstructor(access = lombok.AccessLevel.PRIVATE)
+public class EdcPropertyUtils {
```

Comment:
class should be merged with AssetPropertyMapper. AssetPropertyMapper which handles generic mapping between `Map<String, String>` and `Map<String, Object>` can be renamed to `EdcPropertyUtils`